### PR TITLE
influxdb2: init at v2.0.2

### PR DESCRIPTION
**Pull request by danderson**
Signed-off-by: David Anderson <dave@natulte.net>

###### Motivation for this change

InfluxDB 2.0 is GA upstream.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
This is the PR for the revert https://github.com/NixOS/nixpkgs/commit/fbc4af5725bb8e3449ac84bffb870a3f4047220b and the PR https://github.com/NixOS/nixpkgs/pull/104484

**Comment by danderson** [comment]
Yup. I think I corrected the ofborg failure, but I'm waiting for the CI verdict to be sure.

File: pkgs/servers/nosql/influxdb2/default.nix [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{ buildGoModule
+, buildGoPackage
+, fetchFromGitHub
+, go-bindata
+, lib
+, llvmPackages
+, mkYarnPackage
+, pkg-config
+, rustPlatform
+, ... }:
```
Comment (cole-h) [inline pkgs/servers/nosql/influxdb2/default.nix:10]: Is the ellipsis necessary?

Comment (danderson) [reply pkgs/servers/nosql/influxdb2/default.nix]: Huh, I guess not! Force of habit from my NixOS configs. Removed.

**Comment by danderson** [comment]
@offlinehacker @zimbatm ready for review. Same PR as last time, except it passes ofborg this time :)

**Pull request merged**